lib/containers/container_images.pm

```diff
@@ -56,9 +56,9 @@ sub build_container_image {
     assert_script_run("$runtime images");
 }
 
-=head2 test_containered_app
+=head2 test_built_image
 
- test_containered_app($runtime, [$buildah], $dockerfile, $base);
+ test_built_image($runtime, [$buildah], $dockerfile, $base);
```

Comment:
if we always pass same Docker file , maybe we can simplify function and hardcode Dockerfile ?